lib/europeana/feedback_button.rb

```diff
@@ -1,8 +1,35 @@
 # frozen_string_literal: true
+
 require 'europeana/feedback_button/engine'
 
 module Europeana
   module FeedbackButton
+    # TODO: why is this necessary?
```

Comment:
This doesn't seem to be necessary. Removing it 'worked' the same as with this present.